server/planning/assignments_content.py

```diff
@@ -40,6 +46,15 @@ def get_item_from_assignment(assignment):
 
     ednote = (assignment.get('planning') or {}).get('ednote')
 
+    planning_item = assignment.get('planning_item')
+    # we now merge planning data if they are set
+    if planning_item is not None:
+        planning = superdesk.get_resource_service('planning').find_one(req=None, _id=planning_item)
+        if planning is not None:
+            for field in FIELDS_TO_COPY:
+                if field in planning and planning[field]:
+                    item[field] = planning[field]
```

Comment:
When we're creating a Planning item from a News Item, we derive `planning.description_text` from either `item.abstract` or `item.headline` (whichever has a value).
I would expect when creating a News Item that `item.headline` would be derived from `planning.description_text`

Reply:
OK I have changed it.